src/CLI/OptionsEvaluator.cs

```diff
@@ -18,7 +19,7 @@ public static IOptions ProcessInputs(Options rawInputs, IProcessHelper processHe
 
             if (delayInSeconds < 0 || delayInSeconds > 60)
             {
-                throw new ParameterException(string.Format(CultureInfo.InvariantCulture, "Invalid delay: {0}. Please enter an integer value from 0 to 60.", delayInSeconds));
+                throw new ParameterException(string.Format(CultureInfo.CurrentCulture, DisplayStrings.ErrorInvalidDelayFormat, delayInSeconds));
```

Comment:
Any user-displayable messages will need CurrentCulture if/when localization is added, so switching to them now